package/system/signify/Makefile

```
@@ -0,0 +1,51 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=signify
+PKG_VERSION:=29
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/aperezdc/$(PKG_NAME)/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=8d255ad88290c9b675711f47cc7336b736f6a7adcccd822a0c1a03e3d53de41c
+
+PKG_LICENSE:=ISC
+PKG_LICENSE_FILES:=
+
+PKG_MAINTAINER:=Paul Spooren <mail@aparcar.org>
+
+PKG_BUILD_PARALLEL:=1
+HOST_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+
+HOST_BUILD_DEPENDS:=libbsd/host
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
+
+TARGET_CFLAGS += -ffunction-sections -fdata-sections
+TARGET_LDFLAGS += -Wl,--gc-sections
+# MAKE_VARS += VERIFY_ONLY=1
+
+define Package/signify
+  SECTION:=base
+  CATEGORY:=Base system
+  DEPENDS:=+libbsd
+  TITLE:=OpenWrt signature verification utility
+endef
+
+define Package/signify/description
+  OpenBSD tool to signs and verify signatures on files. Portable version.
+endef
+
+define Package/signify/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/signify $(1)/usr/bin/signify
+endef
+
+define Host/Install
+        $(INSTALL_BIN) $(HOST_BUILD_DIR)/signify $(STAGING_DIR_HOST)/bin/signify
```

This has different indentation than what is in `define Package/signify/install`.

```suggestion
	$(INSTALL_BIN) $(HOST_BUILD_DIR)/signify $(STAGING_DIR_HOST)/bin/signify
```